docs/guides/networking/linode-network/ip-failover/index.md

```diff
@@ -215,20 +224,18 @@ If desired, both instances can be configured with the same role (both primary or
 
 You can test the failover functionality of the shared IP using the steps below.
 
-1.  Power off the *primary* Compute Instance.
-
-1.  Using your local machine, ping the shared IP address.
+1.  Using a machine other than the two Compute Instances within the failover configuration (such as your local machine), ping the shared IP address.
 
         ping 192.0.2.1
```

Comment:
@wildmanonline maybe we should put a placeholder `[shared-ip]` here instead of the example IP (192.0.2.1)?

Reply:
I like the way you think. I'll commit that now.